src/components/SidebarNav/SidebarNavLink.tsx

```diff
@@ -0,0 +1,23 @@
+import { NavLink } from "react-router-dom";
+
+interface SidebarNavLinkProps {
+  route: string;
+  displayText: string;
+}
+
+export function SidebarNavLink(props: SidebarNavLinkProps): JSX.Element {
+  let activeStyle =
```

Comment:
consider making these variables `const`s. We should configure the lint rules to warn you of this.